.travis.yml

```diff
@@ -1,14 +1,22 @@
 language: julia
-julia: 1.1
-os: linux
```

Comment:
this is not the right way to ask travis to run tests and, therefore, travis was only building the docs

Reply:
you mean, the syntax was wrong? thanks for fixing it.

Reply:
Yes that’s what I meant 🤙🏼